server/server.go

```diff
@@ -233,15 +250,27 @@ func New(config *Config) (*Server, error) {
 
 	utils.StartReaper()
 
-	if err := os.MkdirAll(config.ImageDir, 0755); err != nil {
+	store, err := storage.GetStore(storage.StoreOptions{
+		RunRoot:            config.RunRoot,
+		GraphRoot:          config.Root,
+		GraphDriverName:    config.Storage,
+		GraphDriverOptions: config.StorageOption,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	imageService, err := scri.GetStorageImageService(store, config.DefaultTransport)
```

Comment:
We could.  I think it would actually be cleaner in terms of dependencies to move the entirety of `containers/storage/cri` into this repo, but I also want it to be in a place where other projects and reuse the logic instead of having to reimplement it themselves, and I'm not sure that's within the scope of what this repo wants to be.

Turning the image functions into members of an interface that isn't exactly `runtime.ImageServiceServer` would let us move some of its API glue here, and let `containers/storage` lose its hard dependency on `k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime`, which pulls in quite a few other things, so maybe that'd work better.